sdk/cosmosdb/cosmos/src/client/Container/Containers.ts

```diff
@@ -134,7 +134,7 @@ export class Containers {
       };
     }
 
-    const response = await this.clientContext.create<ContainerRequest>({
+    const response = await this.clientContext.create<ContainerRequest, ContainerDefinition>({
```

Comment:
What changed that made it required to specify ContainerDefinition?

Reply:
requests can specify partitionKey with a string (you can see how it gets expanded above before the request is made.) 

The server, however, responds with the object form, which we expect. The `create` wrapper defaults to having the input shape match the output shape... which isn't quite correct as we know we've normalized partitionKey already.

The mismatch in types of partitionKey was blowing up when we construct the  `ContainerResponse` below.

Reply:
Ah, so TypeScript was complaining as it should have been already?

Reply:
surprised this wasn’t complaining before, but the change and explanation is right